app/views/user_mailer/invite.html.erb

```diff
@@ -0,0 +1,11 @@
+<% u = @user %>
+
+<p>
+  Welcome!
+</p>
+<p>
+  You've been invited to contribute to the Baltimore Public Art Commons. Please click the following link to complete your registration.
+</p>
+<p>
+  <%= link_to "Finish Registration", edit_user_url(u, token: @token) %>
```

Comment:
I'm not sure there's an advantage to defining `@token` if you can call `@user.token`.